.gitignore

```diff
@@ -13,6 +13,7 @@ toolkit.xml
 /com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/*/*.xml
 /com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/*/*.gif
 !com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/native.function/function.xml
+!com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/HTTPRequestAsync/HTTPRequestAsync.xml
```

Comment:
In general used librarys should be downloaded during build process and should not be copied into the repository. 
should do something with wget or similar to download during build and add a link to the licence or copy the licence.

Reply:
This is for C++ operator model, not for libraries (as you remember, you suggested to add it because as opposed to Java operator model being generated, the C++ operator model is persistent).
For libraries there is an `ant` file to download and build them.